src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $color-bg-primary;
+  border: 1px solid #f3f3f3;
+
+  &__media {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px;
+  }
+
+  &__content {
+    padding: 8px 16px 16px;
```

Comment:
```suggestion
    padding: 8px 16px;
```

Reply:
I need these padding, as changing the value to two will disrupt the layout conditions.

Reply:
:+1: my bad - keep it as it is